File: lab/python/matematica_e_operadores/precedencia.py
def calcular_expressao1(x, y, z):
    return x + y * z


def calcular_expressao2(x, y, z):
    return (x + y) * z


def verificar_se_expressao_e_verdadeira(x, y, z):
    return 1 if (x > y) and (y < z) else 0


n1 = int(input("N1: "))
n2 = int(input("N2: "))
n3 = int(input("N3: "))

print(calcular_expressao1(n1, n2, n3))
print(calcular_expressao2(n1, n2, n3))
print(verificar_se_expressao_e_verdadeira(n1, n2, n3))

File: lab/javascript/matematica_e_operadores/precedencia.js
function calcular_expressao1(x, y, z){
    return x + y * z;
}


function calcular_expressao2(x, y, z){
    return (x + y) * z;
}


function verificar_se_expressao_e_verdadeira(x, y, z){
    if ((x > y) && (y < z)){
        return 1;
    }
    return 0;
}

const prompt = require('prompt-sync')();
let x = parseFloat(prompt("Digite o valor de x: "));
let y = parseFloat(prompt("Digite o valor de y: "));
let z = parseFloat(prompt("Digite o valor de z: "));

console.log(calcular_expressao1(x, y, z));
console.log(calcular_expressao2(x, y, z));
console.log(verificar_se_expressao_e_verdadeira(x, y, z));
